pkg/client/automation/client.go

```diff
@@ -120,6 +120,37 @@ func WithCustomUserAgentString(userAgent string) func(client *Client) {
 	}
 }
 
+// Get returns a single automation object matching the given resource type and id or an error
+// if a resource with the given ID and resourceType wasn't found
+func (a Client) Get(ctx context.Context, id string, resourceType ResourceType) (result *Response, err error) {
```

Comment:
nit: having the resourceType before the ID would make more sense to me, listing them in a hierarchical way, and matches the (API, id) params of the config client:
```suggestion
func (a Client) Get(ctx context.Context, resourceType ResourceType, id string) (result *Response, err error) {
```

Reply:
yeah why not :)